src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -129,9 +129,13 @@ private void testEditSessionLink() {
     private void testEditSessionAction() throws Exception {
 
         ______TS("typical success case");
+        
+        feedbackEditPage.clickStartDateBox();
+        assertTrue(feedbackEditPage.isDatesOfPreviousCurrentAndNextMonthEnabled());
 
         feedbackEditPage.clickEndDateBox();
-        assertTrue(feedbackEditPage.verifyEndDatesBeforeTodayAreDisabled());
+        assertTrue(feedbackEditPage.isDatesOfPreviousCurrentAndNextMonthEnabled());
```

Comment:
This test does not fail even on master. The end date used in this test is in the future, although this issue is about being able to set end dates in the past.